.github/ISSUE_TEMPLATE/feature-request.yaml

```diff
@@ -0,0 +1,53 @@
+name: Feature Request
+description: A feature request for WildFly
+title: ""
+labels: ["feature"]
+projects: ["wildfly/7"]
+body:
+  - type: input
+    id: jira
+    attributes:
+      label: JIRA
+      description: A link to the main feature request JIRA.
+      placeholder: ex. https://issues.redhat.com/browse/WFLY-XXX
+    validations:
+      required: true
```

Comment:
Is it really required to propose a new feature enhancement?

It's needed for the analysis document but for proposing something that might be too early. It might not be necessarily known beforehand which projects would be impacted by the proposal (WFLY, WFCORE, WFMP, etc.)

Reply:
No, this doesn't really need to be required I suppose. The idea here is we use this for RFE's that are effectively already in progress. This will not be meant to track features we want to have, just features we're actively working on for a release.

Reply:
This data is necessary in the proposal, so repeating it here is added overhead.